### PR TITLE
feat: add terminal-sized Electra giraffe banner

### PR DESCRIPTION
**Pull request by ClockworkYuzu**
**Motivation**

Alternative to #7233 by shrinking the banner to the standard linux terminal size (80 columns and 24 rows)

**Evidence**

Tested on several window sizes with MBP13 with default resolution:

80x24
Screenshot 2024-12-06 at 8 41 27 AM

215x32(Half horizontal screen)
Screenshot 2024-12-06 at 8 37 23 AM

215x64(Full screen)
Screenshot 2024-12-06 at 8 32 45 AM

107x64(Half vertical screen)
Screenshot 2024-12-06 at 8 31 37 AM

### CODE REVIEW

File: packages/beacon-node/src/chain/blocks/utils/giraffeBanner.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export const GIRAFFE_BANNER = String.raw`
```
Comment (nflaig) [inline packages/beacon-node/src/chain/blocks/utils/giraffeBanner.ts]:
```suggestion
export const ELECTRA_GIRAFFE_BANNER = String.raw`
```

**Review by nflaig — COMMENTED** [review]
Thanks @this-is-yvette, that's much better :giraffe:

**Review by nflaig — APPROVED** [review]

**Comment by wemeetagain** [comment]
sweeeeeet

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.24.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.24.0) :tada:

**Pull request merged**